zulipterminal/version.py

```diff
@@ -1 +1,8 @@
 ZT_VERSION = '0.5.1+git'
+
+SUPPORTED_SERVER_VERSIONS = [
+    ('2.1', None),
+    ('2.2', 1),
```

Comment:
Would 1, as a feature level, be sufficient for now or should I extend it till 14 (the latest feature level)?

Reply:
Any finite value seems ok for now, but we should add a comment given that 2.2 is not actually yet released!